src/EnergyPlus/Plant/Component.cc

```diff
@@ -54,15 +54,15 @@ namespace DataPlant {
 
     bool CompData::isPump()
     {
-        if (this->TypeOf_Num == DataPlant::TypeOf_PumpConstantSpeed) {
+        if (this->Type == DataPlant::PlantEquipmentType::PumpConstantSpeed) {
```

Comment:
Entire function can be replaced by `constexpr std::array<bool> PlantEquipmentTypeIsPump;`